app/public/cantusdata/management/commands/index_manuscript_mei.py

```diff
@@ -1,21 +1,30 @@
 from typing import Any, Dict
 from os import path, listdir
+import re
 
 from django.core.management.base import BaseCommand, CommandParser
 from django.conf import settings
+from solr.core import SolrConnection  # type: ignore
```

Comment:
```suggestion
from solr.core import SolrConnection
```

Reply:
Oh, I just didn't know this was related to the type checker, I just thought it was an erroneous comment! 

I'm not yet using type checker for CantusDB, so I should set that up. It's okay to keep as is.

Reply:
Ok! I will add the more specific ignore then, but keep the comment in! Thanks for checking.